src/components/FileList/FileElement.js

```diff
@@ -86,57 +64,47 @@ const FileElement = ({
       ? fileTypeDefault[fileNode.fileType]
       : 'unknown';
   return (
-    <li className={fileElementClass}>
-      <div className="FileList--ElementBadge sm:w-48">
-        <Badge state="new" loading={loading} className="sm:min-w-full" />
-        <Badge
-          state="pendingApproval"
-          loading={loading}
-          className="sm:min-w-full"
-        />
-      </div>
-      <div className="flex-initial">
-        <p className="m-0">
-          <span className={fileNameClass} title={fileDescription}>
-            {fileName}
-          </span>
+    <li className={fileElementClass} tabindex="0">
+      <GridContainer fullWidt className="ml-0 mr-0 p-0">
+        <div className="cell-2 row-1 h-full border-r border-grey-lightest text-center">
+          <Badge state="pendingApproval" loading={loading} />
+        </div>
+
+        <div className="cell-9 h-full">
+          <Link
+            to={`/study/${match.params.kfId}/files/${fileKfID}`}
+            className="w-full no-underline text-black inline-block"
+          >
+            <p className={fileNameClass}>{fileName}</p>
+            {fileDescription ? (
+              <p className="FileList--Element-desc w-3/4 h-8 m-0 p-0 text-truncate text-black">
```

Comment:
`FileList--Description`?

Reply:
that would mean it's describing the `FileList` whereas this really describes the `FileList--Item` or `Element`